### PR TITLE
Feat/add user authentication

### PR DESCRIPTION
**Pull request by ImMPrada**
This PR enables a user to sign up, in and out

### CODE REVIEW

File: app/controllers/clubhouses_controller.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class ClubhousesController < ApplicationController
+  before_action :authenticate_user!, only: %i[new create]
```
Comment (mapra99) [inline app/controllers/clubhouses_controller.rb:2]: according to this a non-logged in  user will be able to see the clubhouses. Is that right?

File: app/controllers/users/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    # before_action :configure_sign_in_params, only: [:create]
+
+    # GET /resource/sign_in
+    # def new
+    #   super
+    # end
+
+    # POST /resource/sign_in
+    def create
+      puts 'TEA'
```
Comment (mapra99) [inline app/controllers/users/sessions_controller.rb]: 👻

File: app/controllers/users/sessions_controller.rb [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    # DELETE /resource/sign_out
+    # def destroy
+    #   super
+    # end
+
+    protected
+
+    def configure_sign_in_params
+      devise_parameter_sanitizer.permit(:sign_in, keys: [:identificator])
```
Comment (mapra99) [inline app/controllers/users/sessions_controller.rb]: The right word in English for this is `identifier`
```suggestion
      devise_parameter_sanitizer.permit(:sign_in, keys: [:identifier])
```

File: app/views/users/registrations/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  <%= form_with(model: @user,
+                 data: {
+                  url: registration_path(@user),
+                  turbo: false
+                 },
+                 class: "min-w-full mt-2") do |f| %>
+
+    <div class="field">
+      <%= f.label :username, 'username:' %>
+      <%= f.text_field :username, { placeholder: UsersController::USERNAME_PLACEHOLDER, class: 'min-w-full' } %>
```
Comment (mapra99) [inline app/views/users/registrations/new.html.erb]: I think controllers shouldn't be used to store any information about the views. If you really need to save this somewhere I think Rails helper modules are a better fit
https://www.rubyguides.com/2020/01/rails-helpers/

File: app/controllers/clubhouses_controller.rb (hunk 1 above)
Comment (ImMPrada) [reply app/controllers/clubhouses_controller.rb:2]: yeah!

File: app/controllers/users/sessions_controller.rb (hunk 2 above)
Comment (ImMPrada) [reply app/controllers/users/sessions_controller.rb]: Tato Estuvo Aquí jajajaja

File: app/controllers/users/sessions_controller.rb (hunk 3 above)
Comment (ImMPrada) [reply app/controllers/users/sessions_controller.rb]: Done! :$

File: app/views/users/registrations/new.html.erb (hunk 4 above)
Comment (ImMPrada) [reply app/views/users/registrations/new.html.erb]: Done!

File: app/controllers/users/sessions_controller.rb (hunk 2 above)
Comment (mapra99) [reply app/controllers/users/sessions_controller.rb]: You should remove debugging logs before merging

**Review by mapra99 — APPROVED** [review]

**Pull request merged**